gui/default/assets/css/overrides.css

```diff
@@ -56,6 +56,10 @@ ul+h5 {
     word-wrap:break-word;
 }
 
+.text-italic {
```

Comment:
> Regarding the MDN description, I think this usage fits alright.

I really wouldn't say so. As I see it, `i` here is just an example of using a specific element purely for its default presentation. On the other hand, `style` is just an attribute, so there's nothing really semantic about it, but the same can be said about all the `ng-*` attributes and such.

Which case described at the MDN would you say this particular sentence fits?

Reply:
> This would be a range of text with different semantic meaning than the surrounding text. Among the use cases for the <i> element are spans of text representing a different quality or mode of text, such as: 
> * Alternative voice or mood

The "surrounding text" are factual numbers, whereas the span in question is a remark about why they might differ.  That's a "different quality" in my understanding.

Anyway, my main concern is code readability here.  The original intent was to make it an `<i>` element, which is usually italicized.  You want to explicitly force it to italics, just with more code.  Same result, more reading.

Reply:
I still think it's a stretch, but I'm not going to argue about the `i` 😉. Everything should be fixed now.